templates/20-validation-webhook.deployment.yaml.tmpl

```diff
@@ -15,15 +15,15 @@ spec:
     spec:
       serviceAccountName: "#SANAME#"
       initContainers:
-      - image: "#IMG#:#IMAGETAG#"
```

Comment:
Does this work? I'm guessing the SelectorSyncSet template will do the substitution?

Reply:
The easiest answer seemed to be updating the template to ${IMAGE_TAG}, though I think it could be cleaned up since we have several variables like IMAGETAG and  IMAGE_TAG that could be done better. The selectorsyncset.yaml file that's in the PR is the resulting output of the changes, including this one.